src/main/java/seedu/cafectrl/CafeCtrl.java

```diff
@@ -66,16 +59,11 @@ private void run() {
                 ui.printLine();
             }
         } while (!command.isExit());
-
-        try {
-            menuStorage.storeData();
-        } catch (IOException e) {
-            ui.showToUser(e.getMessage());
-        }
+        //this.storage.saveAll(this.menu, this.orderList, this.pantry);
     }
 
-    public static void main(String[] args) {
-        CafeCtrl cafeCtrl = new CafeCtrl("data/menu.txt");
+    public static void main(String[] args) throws IOException {
```

Comment:
I am planning to implement it in the next PR (cuz I no time to do it yet) so that you all can use the skeleton file to implement your encoding and decoding while waiting for my next PR

Reply:
or do you think i should just implement it first? I can do it by tmr night

Reply:
its ok you can do it in the next PR, I think leave it as it is for now so that the rest can start working on it if they plan to do it tomorrow